src/components/SettingsButton/SettingsButton.js

```diff
@@ -46,37 +49,57 @@ class SettingsButton extends Component<Props, State> {
     this.setState({ hovered: false });
   };
 
-  handleShowModal = () => {
-    const { settingsFor, showModal } = this.props;
-    showModal(settingsFor);
+  handleClick = () => {
+    const { showProjectSettings } = this.props;
+
+    // NOTE: If we support app settings, use the `settingsFor` prop here to
+    // select which modal to open.
+    showProjectSettings();
   };
 
   render() {
     const { hovered } = this.state;
+
     return (
-      <Motion
-        style={{
-          rotations: spring(hovered ? 0.3 : 0),
-          scale: spring(hovered ? 1.3 : 1),
+      <Spring
+        native
+        config={{
```

Comment:
The animation is not rotating smoothly.
I think it looks better if we would use `import { config } from 'react-spring';` and then `<Spring config={config.slow} .../>` (`config.stiff` is a bit faster and also looks good.)

Reply:
Hm, I wonder if this is just a matter of subjective preference, or if it just doesn't look as good on other computers?

There were two things I wanted to capture:
- I wanted it to be faster, since I think hover effects should be quick in general
- I liked the idea of it being springy (so it would wobble a bit), because it feels a bit playful and organic)

But yeah, if it doesn't look smooth/intentional, that's a problem. I'll play with it, see if I can get something a bit smoother